### DISCUSSION TITLE
Option to show breadcrumbs above page title in subpages

### DISCUSSION THREAD

**Discussion by benzitney** [GitbookIO/community#287] [Feature Requests]
For subpages, it would be great to have the option to show breadcrumbs at the top of the page, above the title, so that the user has context into what the subpage is about. 

For example, in our gitbook site, we have a page for each integration, and then a set of subpages for each integration that are all named the same. We have the same set of subpages for each integration, and we name them generically. 

For example, we will have a page & subpage structure like this:
- Integrations
  - Facebook Ads
    - Integration Setup
    - Integration Specifications
    - Important Notes
  - Google Ads
    - Integration Setup
    - Integration Specifications
    - Important Notes

The problem with this is when we send a link to the user for the Facebook Ads > Integration Setup page, for example, the page heading is generic — "Integration Setup" — and you can't immediately tell that you're on the specific integration setup page for Facebook Ads.

It would be great if we had the option to show breadcrumbs above the title to give the user more immediate context on where they're at. Something like this:
![image](https://github.com/GitbookIO/community/assets/12058457/2afa2814-5087-4fff-9dea-1af310f6920a)

This would only show up if you're on a subpage, since top-level pages would just have the title as the breadcrumb list and would be redundant without providing value

**Comment by schaudhry11** [comment]
This is something we need as well and would be an excellent feature.

**Comment by denisemcg** [comment]
This was released last month:
https://docs.gitbook.com/changelog/december-2024/17-december-visual-upgrades-table-improvements-and-more#site-level-breadcrumbs
I wish breadcrumbs (at least the top level) could be added to the AI search results list for the same reason as above. Often we have the same titled page such as 'Export data' in multiple products but in the search results it's not possible to tell which page belong to which product.
